### ISSUE TITLE
Error while importing Spacy in Windows with Anaconda

### ISSUE THREAD

**Issue by scarescrow** [explosion/spaCy#1199]
Hi,

I have a windows server 2008 R2 (64-bit) machine, where I have Anaconda 4.2.0 (Python 2.7.12 - Winx64) installed.
I tried installing spacy as specified in the docs, using:

`conda install -c conda-forge spacy=1.8`

Install is successful. However, when I open a python shell, and type `import spacy`, I get the following stack trace:

````
>>> import spacy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Anaconda\lib\site-packages\spacy\__init__.py", line 7, in <module>
    from .deprecated import resolve_model_name
  File "C:\Anaconda\lib\site-packages\spacy\deprecated.py", line 4, in <module>
    from .cli import download
  File "C:\Anaconda\lib\site-packages\spacy\cli\__init__.py", line 5, in <module
>
    from .train import train, train_config
  File "C:\Anaconda\lib\site-packages\spacy\cli\train.py", line 8, in <module>
    from ..scorer import Scorer
  File "C:\Anaconda\lib\site-packages\spacy\scorer.py", line 5, in <module>
    from .gold import tags_to_entities
ImportError: DLL load failed: The application has failed to start because its si
de-by-side configuration is incorrect. Please see the application event log or u
se the command-line sxstrace.exe tool for more detail.
````

I tried two more things:
* Tried installing spacy 1.7 instead of 1.8
* Built spacy natively, by downloading the source, and using `python setup.py build`, and copied the resulting output into my site-packages directory.
Both of these result in the same error.

Is this a known issue? Can someone please help?

## My Environment

This is the output of `conda info`:

````
Current conda install:

               platform : win-64
          conda version : 4.3.22
       conda is private : False
      conda-env version : 4.3.22
    conda-build version : 2.0.2
         python version : 2.7.12.final.0
       requests version : 2.13.0
       root environment : C:\Anaconda  (writable)
    default environment : C:\Anaconda
       envs directories : C:\Anaconda\envs
                          C:\Users\Administrator\AppData\Local\conda\conda\envs
                          C:\Users\Administrator\.conda\envs
          package cache : C:\Anaconda\pkgs
                          C:\Users\Administrator\AppData\Local\conda\conda\pkgs
           channel URLs : https://conda.anaconda.org/spacy/win-64
                          https://conda.anaconda.org/spacy/noarch
                          https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/win-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
            config file : C:\Users\Administrator\.condarc
             netrc file : None
           offline mode : False
             user-agent : conda/4.3.22 requests/2.13.0 CPython/2.7.12 Windows/7
Windows/6.1.7601
          administrator : True
````

And this is the output of `conda list spacy`:

````
# packages in environment at C:\Anaconda:
#
spacy                     1.7.3           np111py27_vc9_0  [vc9]  conda-forge
````

**Comment by vascobailao** [comment]
Having the same mistake... Trying to install in a windows 10 laptop

**Comment by scarescrow** [comment]
I found that issue is only reproducible in a setup where Visual C++ is not installed, and it uses the msvcrt90.dll that ships with Anaconda. It seems that when importing the gold.pyd file, it is looking for a DLL Manifest, that is not present, and hence the dll(pyd) fails to load.

**Comment by ines** [comment]
Hoping to finally fix this problem in the new version, v2.0. We're currently waiting to get it built for conda (it's just taking a bit longer because of Travis errors and the OSX backlogs 😓). 

Once the version is up, could you try again? [See here](https://spacy.io/usage/v2) for details on what's new in v2.0. If there are still problems with conda install and the new version, feel free to reopen.
